### ISSUE TITLE
Client loses focus after popup from TV

### ISSUE THREAD

**Issue by Lampan-git** [jellyfin/jellyfin-tizen#311]
**Bug Description**
After a popup occurs from the TV when either using Quick Connect or failing to connect to a server the client loses focus. This removes the blue background from the select so you don't know what you are pressing. Input from the on-screen keyboard doesn't seem to work, though backspace does seem to work, so maybe it's just invisible when writing?

**Reproduction steps**
1. a) Fail to input a valid server in the initial setup
    b) Press the Quick Connect login on the login page
2. Press Ok on the pop up.

**Current Behavior**
No blue background on the selected element, so you don't know what you are going to press. 
Scrolling etc in the home menu after logging in doesn't work, though pressing elements works so you could go below the screen and press something. 

**Expected behavior**
Blue background of the current element choice, scrollable home screen etc.

**Server version**
10.10.1

**Client Version**
Jellyfin-web 10.10.1

**Environment**
TV: S90C
OS: OneUI/Tizen 8, v. 2013

**Comment by Jacob-077** [comment]
I'm having this exact issue. A quicker way to reproduce this is to try to delete any media, once you select cancel you'll lose control of the entire app.

**Comment by mihalyr** [comment]
I've installed finally Jellyfin on the Samsung TV - this repo and the instructions were super helpful and made the entire thing fairly easy. But the first thing I noticed was that when trying to connect to my server and typing in the address, when I wrote the wrong address and a pop-up showed up, next thing I could not control the app with the remote anymore. I could only press back and OK on some option I could not see. So I was able to exit the app and then reopen to try it again. It happened to me several times while I figure out that simple mDNS name won't work and needed the IP, then figured I needed also the port (while on my laptop and mobile I can just use the mDNS name without port - probably a limitation of the TV).

**Comment by dantesbr** [comment]
The same here, my TV is a QN90C with Tizen 8 version 2011

**Comment by morpheus133** [comment]
Just for helping the investigation:
On Tizen 4.0 TV I was not able to reproduce it.
Also on both emulators(including 8.0) it seems working.

**Comment by dantesbr** [comment]
> Just for helping the investigation: On Tizen 4.0 TV I was not able to reproduce it. Also, on both emulators (including 8.0), it seems to be working.  

Yes, the issue does not occur on the emulator, only on the TV itself.  
I managed to fix it by editing the source code in `jellyfin-web/src/components/confirm/confirm.js`:  

```javascript
function useNativeConfirm() {
    // webOS seems to block modals
    // Tizen 2.x seems to block modals
    // return !browser.web0s
    //     && !(browser.tizenVersion && browser.tizenVersion < 3)
    //     && browser.tv
    //     && window.confirm;
    return false;
}
```

**Comment by morpheus133** [comment]
@dantesbr will you open a PR for the correction?
